src/icons/later_today.svg

```diff
@@ -3,7 +3,7 @@
 <svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
 	 viewBox="0 0 24 24" style="enable-background:new 0 0 24 24;" xml:space="preserve">
 <style type="text/css">
-	.st0{fill:#53C942;}
+	.st0{fill:#30e60b;}

```

Comment:
I guess we can't use the `photon-colors` variables here, but could we at least add a comment indicating which one it is, to make it easier to upgrade?

Reply:
Also, we seem to have standardized on uppercase hex colors.